code/src/index.css

```diff
@@ -1,13 +1,186 @@
+
+
```

Comment:
Just remember to clean the CSS up a bit for readability. Remove empty spaces in some paces and add a break in other so it's easier to read.